.github/workflows/pr_autoupdate.yaml

```diff
@@ -1,7 +1,7 @@
 name: Run linters
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
otherwise `pre-commit-ci/lite-action` wont run, it's [hardcoded here](https://github.com/pre-commit-ci/lite-action/blob/5d6cc0eb514c891a40562a58a8e71576c5c7fb43/main.mjs#L8)